Wireframe/index.html

```diff
@@ -4,10 +4,62 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="Get to know what is Git.">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <h1> Get to know Git</h1>
+    </header>
+    <main class="main">
+<!-- What is Git? -->
+      <section class="introduction">
+        <figure>
+          <img src="images/GIT-datascientest.jpg" alt="git picture"> 
+          <figcaption class = "caption">
+            <h2>What is Git? </h2>
+            <p>Git is a powerful version control system that helps developers manage and track changes in their code. 
+              It was created by Linus Torvalds in 2005 to support the development of the Linux kernel. Unlike traditional version control systems, Git is distributed, 
+              meaning that every developer has a complete copy of the entire repository, including its history, on their local machine.</p>
+            <a href ='https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F'>
+              <button>Read More</button>
+            </a>
+          </figcaption>
+        </figure>
+      </section>
+      <section class="two-blocks">
+<!-- Why do developers need Git? -->
+        <article class="benefits">
+          <img src="images/git-benefits.png" alt="git benefits" class="benefits-pic"> 
+          <figcaption class = "caption">
+            <h2>Why do developers need Git? </h2>
+            <p>One of the biggest advantages of Git is its branching capabilities.  Git branches are cheap and easy to merge.
+              Feature branches provide an isolated environment for every change to your codebase.</p>
+            <a href="https://www.atlassian.com/git/tutorials/why-git">
+              <button>Read More</button>
+            </a>
```

Comment:
It is recommended not to have a button tag inside anchor tag
```suggestion
<a href="https://www.atlassian.com/git/tutorials/why-git">Read More</a>